IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SectionInsulator/DocPropertySet.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SectionInsulator" UniqueId="cb4e84ae-2265-3263-b595-78cae54076f8" ApplicableType="IfcDiscreteAccessory/INSULATOR" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="RatedVoltage_0KkDpffYz8L86gzG31TCSi" />
+		<DocProperty xsi:nil="true" href="ACResistance_2dT1WHiH7Ge2PeqXa73Sp" />
+		<DocProperty xsi:nil="true" href="BreakdownVoltageTolerance_36oHQCC0b0jubTadGmGjOS" />
+		<DocProperty xsi:nil="true" href="NumberOfWires_2vNnC4jfb7dgwh_lRMrBkU" />
+		<DocProperty xsi:nil="true" href="MaximumOperatingVoltage_02pXPLOwL6PwVg4jmeVjI6" />
+		<DocProperty xsi:nil="true" href="IsArcSuppressed_2bj3cZYAD7qvpndHNlg9vP" />
+		<DocProperty xsi:nil="true" href="MaximumDeflection_3GDyoQKz1DDPMnkpxJXV7P" />
+		<DocProperty xsi:nil="true" href="MinimumAirGapDistance_2mybns6p50AwDgAqzOYpqr" />
+		<DocProperty xsi:nil="true" href="MinimumLeakageDistance_25T8FhIC540O926Lkd6LtI" />
```

Comment:
these seem to be worded as design criteria not performance/physical values of the product please address.